arbeitszeit_flask/translations/de/LC_MESSAGES/messages.po

```diff
@@ -1084,98 +1049,109 @@ msgstr "Verkaufsabweichung"
+#: arbeitszeit_web/formatters/plan_details_formatter.py:75
 msgid "Costs for fixed means of production"
 msgstr "Kosten für feste Produktionsmittel"
 
-#: arbeitszeit_flask/templates/macros/draft_form.html:73
-#: arbeitszeit_web/formatters/plan_details_formatter.py:82
+#: arbeitszeit_flask/templates/macros/draft_form.html:74
+#: arbeitszeit_flask/templates/macros/plan_details.html:133
+msgid ""
+"Here, recurring costs from raw materials and consumables are summarized, e.g. "
+"electricity, rent for production halls, maintenance of machines etc."
+msgstr ""
+"Hier sollten wiederkehrende Kosten aus Rohstoffen und Verbrauchsmaterialien "
+"zusammengefasst werden, z.B. Strom, Miete für Produktionshallen, Wartung von "
+"Maschinen usw."
+
+#: arbeitszeit_flask/templates/macros/draft_form.html:76
+#: arbeitszeit_web/formatters/plan_details_formatter.py:79
 msgid "Costs for liquid means of production"
 msgstr "Kosten für flüssige Produktionsmittel"
 
-#: arbeitszeit_flask/templates/macros/draft_form.html:80
+#: arbeitszeit_flask/templates/macros/draft_form.html:84
+#: arbeitszeit_flask/templates/macros/plan_details.html:140
+msgid "Planned hours of human work."
+msgstr "Geplante Stunden menschlicher Arbeit."
+
+#: arbeitszeit_flask/templates/macros/draft_form.html:86
 msgid "Costs for labour"
 msgstr "Kosten für Arbeit"
 
-#: arbeitszeit_flask/templates/macros/draft_form.html:98
-msgid "Specify product name and description."
-msgstr "Gib Produktname und Produktbeschreibung an."
-
-#: arbeitszeit_flask/templates/macros/draft_form.html:99
+#: arbeitszeit_flask/templates/macros/draft_form.html:95
 msgid ""
-"Choose a planning timeframe in which you are planning to produce and "
-"deliver your product."
+"Products and services within a supply chain should not be marked public. Only "
+"if your product should be consumed for free by all members of society - "
+"funded via the Factor of Individual Consumption (FIC) - then please checkmark "
+"this box."
 msgstr ""
-"Wähle einen Planungszeitraum, in dem du planst, dein Produkt herzustellen "
-"und zu liefern."
-
-#: arbeitszeit_flask/templates/macros/draft_form.html:100
-msgid "Estimate what costs you will have in the given timeframe."
-msgstr "Schätze, welche Kosten du im angegebenen Zeitraum haben wirst."
-
-#: arbeitszeit_flask/templates/macros/draft_form.html:101
-msgid ""
-"Describe the smallest delivery unit and what amount of these units you will"
-" produce."
-msgstr ""
-"Beschreibe die kleinste Abgabeeinheit und welche Menge von diesen Einheiten"
-" du produzieren wirst."
-
-#: arbeitszeit_flask/templates/macros/draft_form.html:102
-msgid "The type of plan. Productive plans are the norm."
-msgstr "Die Art des Plans. Produktive Pläne sind die Regel."
+"Produkte und Dienstleistungen innerhalb einer Lieferkette sollten hier nicht "
+"als öffentlich markiert werden. Nur wenn dein Produkt von allen Menschen der "
+"Gesellschaft umsonst konsumiert werden soll -  im Rahmen des Faktors "
```

Comment:
I'm not completely happy with the wording "konsumiert im Rahmen des FIKs". I think I prefered "finanziert über den...", even though "finanziert" is a monetary term. 
Perhaps we should even delete the sentence "funded via the Factor of Individual Consumption (FIC)" from the original - we could explain how public plans work somewhere else.